### ISSUE TITLE
Unable to use loss function 'SoftmaxCrossEntropyOHEMLoss'

### ISSUE THREAD

**Issue by tensionfly** [dmlc/gluon-cv#973]
I import the loss function 'SoftmaxCrossEntropyOHEMLoss' from gluoncv.loss . When i run my code,  an error 'mxnet.ndarray.contrib module has no attribute of SoftmaxOHEMOutput' will be raised. My mxnet and gluoncv are both the newest.

**Comment by ngunauj** [comment]
me too. He makes me very confused. May need an old version of mxnet?
